assets/locales/en.json

```diff
@@ -16,7 +16,7 @@
   "readReleaseNotes": "Read Release Notes",
   "status": "Status",
   "files": "Files",
-  "settings": "Settings",
```

Comment:
Removing this from the top level menu may be a problem.  I believe users will look for the "WebUI Settings" screen under "Settings" and won't find any entry point there. 

Can we add it at the very top, to maintain continuity, of sorts? 

One entry followed by a separator and then existing entries under "Desktop" header (similar to existing  "Experiments"):

> ![2020-04-21--03-12-26](https://user-images.githubusercontent.com/157609/79814286-22221b80-837e-11ea-95b7-f1cbbe8166af.png)


@jessicaschilling  Does this make sense? If so, what would be the correct wording here: "Open Node Settings"? "Open WebUI Settings"?

Reply:
Just rebased on top of master and added a "Open Node Settings" which opens Web UI settings:

![image](https://user-images.githubusercontent.com/5447088/79838130-73480480-83aa-11ea-87f1-9f6efc02c124.png)

Reply:
This is great. Agree on “Node Settings” over “WebUI settings sine the thing you’re adjusting is the node itself.

Reply:
What about adding `Desktop integrations` header below the separator, to make it even more clear?

> 
![png](https://user-images.githubusercontent.com/157609/79870602-83300a80-83e3-11ea-824d-83dd831cb6d4.png)

Reply:
Just added:
![image](https://user-images.githubusercontent.com/5447088/79871106-cedeb600-83db-11ea-8fba-f6f02e3e0451.png)


Just wondering if it's really needed. But looks great

Reply:
Having a clarification of “these are the prefs for this app” is a good idea. I might just say “App Preferences” though? Not sure if “desktop integrations” makes sense unless you’re already deeply familiar.

Reply:
@lidel what do you think?